pandas/tseries/tools.py

```diff
@@ -349,7 +350,11 @@ def _convert_listlike(arg, box, format):
     elif com.is_list_like(arg):
         return _convert_listlike(arg, box, format)
 
-    return _convert_listlike(np.array([ arg ]), box, format)[0]
+    try:
+        return _convert_listlike(np.array([ arg ]), box, format)[0]
+    except ValueError as e:
+        if not coerce:
```

Comment:
coerce is already passed to the cython functions 

my point is that this should be cause and coerced at a lower level